### ISSUE TITLE
`System.Object' is not defined or imported`

### ISSUE THREAD

**Issue by thedumbtechguy** [oleg-shilo/cs-script#80]
I am running dot net core 2.0 and I installed the `CS-Script 3.27.0` nuget package.

No matter what example I try from the wiki, I get error below. I am not sure what I have to do to get it to run.

```csharp
CompilerException: (1,1): error CS0518: The predefined type `System.Object' is not defined or imported
```

**Comment by oleg-shilo** [comment]
The NuGet packages are designed for non-CORE projects. I am surprised the package manager let them to be downloaded to the .NET Core project.

Anyway, if you are not really keen on Core then change the project type to .NET and then you can used the NuGet packages. 

If you indeed need .NET Core then you need to get the CS-Script assembly not from NuGet but from GitHub [releases page](https://github.com/oleg-shilo/cs-script/releases). 

![image](https://user-images.githubusercontent.com/16729806/28604486-48a7f290-720e-11e7-96ad-928e734be5c6.png)

The reasons for this are discussed in this issue: https://github.com/oleg-shilo/cs-script/issues/18. 

Basically .NET Core is still lacking some functionality to allow CS-Script API to be finalized. Thus the current API is not published on NuGet yet but on GitHub only.

**Comment by thedumbtechguy** [comment]
O I see. I'm guessing it's because I'm on net core 2.0. 
Thank you.

**Comment by oleg-shilo** [comment]
>I'm guessing it's because I'm on net core 2.0.

Most likely.

BTW I am planing to release  a NuGet package very soon as a pre-release. The very latest .net core is not as incomplete as it was initially.
